include/lattices/vector_clock_pair_lattice.hpp

```diff
@@ -17,21 +17,23 @@
 
 #include "../lattices/core_lattices.hpp"
 
+using VC = MapLattice<string, MaxLattice<unsigned>>;
```

Comment:
Style nit: Should. be `VectorClock`, not `VC`.

Reply:
will fix

Reply:
Bump.